.github/workflows/conflictcheck.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check if PRs have merge conflicts
-        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        uses: eps1lon/actions-label-merge-conflict@e62d7a53ff8be8b97684bffb6cfbbf3fc1115e2e  # v3.0.0
```

Comment:
I assume using the hash for paranoia reasons?

Reply:
Yes, @matthewfeickert did this elsewhere for us in #26025.

Reply:
There's no real downside to using the hash, but for the record I generally only care about using it when it comes to supply chain security. I'll defer to you all on if https://github.com/eps1lon/actions-label-merge-conflict is critical enough to your infrastructure that it needs hashes or not (they will get autoupdated by Dependabot either way).

Reply:
It's not super privileged, but it does have [PR write permissions](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests).